projects/sol/src/lib/sol.component.spec.ts

```diff
@@ -22,13 +22,29 @@ describe('SolComponent', () => {
     .compileComponents()
   })
 
-  beforeEach(() => {
+  beforeEach(fakeAsync(() => {
     fixture = TestBed.createComponent(SolComponent)
     component = fixture.componentInstance
+    component.mpsServer = 'wss://localhost'
+    component.authToken = 'authToken'
+    tick(4500)
     fixture.detectChanges()
-  })
+    flush()
+  }))
 
   it('should create', () => {
     expect(component).toBeTruthy()
```

Comment:
should create should focus on ensuring all variables are instantiated and defined to what we expect.

all of the following should have an expectation for what they are so the test break if they change:
```
terminal: any
  container!: any
  term: any
  redirector: any
  dataProcessor: any
  deviceState = 0
  logger: ConsoleLogger = new ConsoleLogger(LogLevel.ERROR)
  @Output() deviceStatus: EventEmitter<number> = new EventEmitter<number>()
  @Input() deviceConnection: EventEmitter<boolean> = new EventEmitter<boolean>()
  @Input() public mpsServer = ''
  @Input() public authToken = ''
  @Input() public deviceId = ''
```

Reply:
Updated